packages/website/pages/docs/concepts/bridging.mdx

```diff
@@ -83,27 +83,31 @@ Taiko's bridge utilizes the Signal Service we described. Here is the general use
 2. The Bridge locks the Ether, and stores a message by calling `sendSignal(message)` on the SignalService contract
 3. The user receives Ether on the destination chain, if they (or another) provide a valid merkle proof that the message was received on the source chain
 
-### How does ERC-20 bridging work?
+With the current design there are 2 ways to bridge `Ether`:
+1. `Ether` only case: The user interacts directly with the Bridge contract (currently this is the only one supported by our UI) by calling `sendMessage`
+2. `ERC-XXX` + `Ether` +  case: The user interacts with the `ERCXXXVault` (ERC20, ERC721, ERC1155) because he/she wants to bridge over some tokens, but in case he/she fills the `message.value` field, also `Ether` will be bridged (currently this is not supported by our UI)
```

Comment:
I suggest deleting "(currently this is not supported by our UI)" as we will need to delete it soon after UI supports it.